addons/common/stringtable.xml

```diff
@@ -320,11 +320,11 @@
             <Spanish>Rechazar peticiones</Spanish>
             <Polish>Ignoruj prośby</Polish>
             <Czech>Zamítnout žádost</Czech>
+            <Italian>Rifiuta la richiesta</Italian>
             <Russian>Отклонить запросы</Russian>
             <French>Rejeter requête</French>
             <Hungarian>Kérések elutasítása</Hungarian>
             <Portuguese>Rejeitar pedido</Portuguese>
-            <Italian>Rifiuta la richiesta</Italian>
         </Key>
         <Key ID="STR_ACE_ACTION_ACCEPT_REQUEST_KEY_TOOLTIP">
```

Comment:
Some strings were moved while I was fixing them (because I overwritted them before), I hope it's not a problem. Proper one are restored anyway.